### DISCUSSION TITLE
Hide email subject as well as the sender name when PGP is enabled

### DISCUSSION THREAD

**Discussion by 0xspade** [simple-login/app#895] [Feature Requests]
Hi,

Since there's a feature “Hide email subject when PGP is enabled”, Is it also possible to hide sender name or at least set a Sender Name if the PGP is enabled? I also have a concern  when replying while this feature is enable, the original subject should be the original if I reply to the sender not the one I set if the PGP is enable, did I say it right?

**Comment by nguyenkims** [comment]
> Is it also possible to hide sender name or at least set a Sender Name if the PGP is enabled?

You can set the sender name to only reverse alias so it doesn't reveal any info on the sender.

<img width="1170" alt="image" src="https://user-images.githubusercontent.com/3167780/163356970-982534cc-8b08-4106-8d01-521e58b030d5.png">

>  I also have a concern when replying while this feature is enable, the original subject should be the original if I reply to the sender not the one I set if the PGP is enable, did I say it right?

When you reply to an email, the subject of the previous email should be used which is the generic subject.

**Reply by 0xspade** [reply]
>You can set the sender name to only reverse alias so it doesn't reveal any info on the sender.

It's currently missing in self-hosted version, probably needed to wait for update. :D

>When you reply to an email, the subject of the previous email should be used which is the generic subject.

Is that automatic or do I still need to do something?

**Reply by nguyenkims** [reply]
ah yeah, it's probably not included in the current self-hosted version.

> Is that automatic or do I still need to do something?

It's often the default behavior of a mailbox service.
